en/jobs.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: english-front-page
+title: Jobs
+permalink: /en/jobs
+display_big_header: false
+---
+
+# Jobs
+
+# Go Developer
+
+We’re looking for an experienced Go developer to join our development team. This assignment is for a duration of 1 year.
+
+## About Nuts
+Nuts is an initiative by around 30 parties in the Dutch healthcare domain. We build open standards and software to standardize the tedious work of creating safe connections between electronic health records systems, required to exchange medical information. Every EHR system spins up a local Nuts-node which becomes part of a large and decentralized trust network. Your job will be developing these standards and implementing them in the Nuts-node software.
+
+## What we expect of you
+You’ll be working on open-source software, so you are the type of person that likes to share your work with the world. You like working with decentralized identity frameworks, blockchain-like solutions, peer-2-peer technology and various cryptographic schemes. We’re not just trying to build a healthcare infrastructure, we’re building an infrastructure that is fair, future-proof and scalable! Do you want to work on next-gen technology and do something good for the world at the same time?
+
+Since it’s an assignment for a limited time, you’ll need to hit the ground running. This means you have a lot of experience with the Go programming language and you’re able to build systems from the ground up.
+
+To sum it up, you have extensive experience with the following:
+* The Go programming language
+* Several cryptographic schemes
+* Writing API specifications using OpenAPI Specification (Swagger doc)
+* Writing reusable software specifications (RFCs)
+* Distributed systems in general
+* Docker and basic DevOps
+
+## Your employer
+Since Nuts is a decentralized initiative, the actual position is offered by one of the participating organizations. For this specific position, the offering organization is [Nedap Healthcare](https://nedap.com/): A Dutch company located in Groenlo, the Netherlands.
+
+## Your team
+You'll be joining an open-source community. The majority of the interaction will be with the "core-team". This is a team of several enthusiastic and highly skilled people, each with their own interests. Every member is treated equally and a well-motivated opinion is highly appreciated. Most of the interaction will be online through Slack and Zoom. Most core-members live in the eastern part of the Netherlands and therefore have the ability to meet face-to-face but the team is also perfectly fine with joining you remotely. In order for the team to communicate optimally, it's required you're being able to meet face to face on a daily basis. We encourage you to make your own schedule but keep in mind the core-team is located in the CET time zone.
```

Comment:
`it's required you're being able to meet face to face on a daily basis`

`it's required you're able to meet face to face on a daily basis` << This includes video conferencing???